.circleci/config.yml

```diff
@@ -1,94 +1,72 @@
-            docker run -d -p 3000:3000 --name $CIRCLE_SHA1 -e AUTH0_DOMAIN=$AUTH0_TEST_DOMAIN -e AUTH0_AUDIENCE=$AUTH0_TEST_API_IDENTIFIER $CIRCLE_JOB
-      - run:
-          name: Wait for app to be available
-          command: |
-            sleep 10
-            docker run --network host --rm appropriate/curl --retry 8 --retry-connrefused -v localhost:3000
-      - run: 
-          name: Run tests
-          command: |
-            docker create --network host --name tester codeception/codeceptjs codeceptjs run-multiple --all --steps --verbose
-            docker cp $(pwd)/lock_login_test.js tester:/tests/lock_login_test.js
-            docker cp $(pwd)/codecept.conf.js tester:/tests/codecept.conf.js
-            docker start -i tester
-          working_directory: scripts
-      - run:
-          name: Copy app container logs
-          command: |
-            mkdir -p /tmp/out
-            docker logs $CIRCLE_SHA1 > /tmp/out/app_logs.log
-            docker cp tester:/tests/out /tmp/
-          when: on_fail
-      - store_artifacts:
-          path: /tmp/out
+  steps:
+    - attach_workspace:
+        at: ~/
+    - run:
+        name: Replace Auth0 test credentials
+        command: |
+          echo "{ \"domain\": \"$AUTH0_TEST_DOMAIN\", \"clientId\": \"$AUTH0_TEST_CLIENT_ID\", \"audience\": \"$AUTH0_TEST_AUDIENCE\", \"callbackUrl\": \"http://localhost:3000/callback\" }" > $AUTH0_CFG
+            
+    - run:
+        name: Build pull request
+        command: |
+          docker build -t $CIRCLE_JOB ./$SAMPLE_PATH
+          docker run -d -p 3000:3000 --name $CIRCLE_SHA1 $CIRCLE_JOB
+    - run:
+        name: Wait for app to be available
+        command: |
+          sleep 10
+          docker run --network host --rm appropriate/curl --retry 8 --retry-connrefused -v localhost:3000
+    - run:
+        name: Run tests
+        command: |
+          docker create --network host --name tester codeception/codeceptjs codeceptjs run-multiple --all --steps --verbose
+          docker cp $(pwd)/lock_login_spa_test.js tester:/tests/lock_login_test.js
+          docker cp $(pwd)/codecept.conf.js tester:/tests/codecept.conf.js
+          docker start -i tester
+        working_directory: scripts
+    - run:
+        name: Copy app container logs
+        command: |
+          mkdir -p /tmp/out
+          docker logs $CIRCLE_SHA1 > /tmp/out/app_logs.log
+          docker cp tester:/tests/out /tmp/
+        when: on_fail
+    - store_artifacts:
+        path: /tmp/out
 
 # Jobs and Workflows
 version: 2
-jobs: 
+jobs:
   checkout:
     machine: true
     steps:
       - checkout
       - run: git clone https://github.com/auth0-samples/spa-quickstarts-tests scripts
       - persist_to_workspace:
-          root: ~/ 
+          root: ~/
           paths:
             - project
             - scripts
   01-login:
     machine: true
     environment:
-      - AUTH0_CFG: 01-Login/src/Auth/auth0-variables.js
+      - AUTH0_CFG: 01-Login/auth_config.json
```

Comment:
See my other comment about the location of the `auth_config.json` file, but this path should align with that location (currently it's in `01-Login/src/auth_config.json`